tweeter.py

```diff
@@ -99,7 +99,25 @@ def stats():
 
     return stat_string_1 + stat_string_2 + stat_string_3 + stat_string_4
 
-#"Tweeter Stats - Number of adjectives: <num_adj>, Number of objects: <num_obj>, Number of antecedents: <num_ant>, Number of postcedents: <num_post>"
+# 
+
+@app.route("/reversedtweet")
+def reversedtweet():
+    adj_index = random.randint(0, len(adj_list) - 1)
```

Comment:
Since the code to generate a basic tweet is repeated in `tweet()` it might be better to refactor the code in it and here into a new function (something like `generateTweet()`) that can be called in both functions.